setup.py

```diff
@@ -26,8 +28,14 @@ def reqs(filename):
 TEST_REQS = reqs('test-requirements.txt')
 
 
+with open(os.path.join(os.path.dirname(__file__),
+                       'cloudferry', '__init__.py')) as f:
+    VERSION = re.match(r".*__version__ = '(.*?)'", f.read(), re.S).group(1)
```

Comment:
I've changed to the empty version for devel branch. It works the same as without version.
https://github.com/MirantisWorkloadMobility/CloudFerry/pull/985/files#diff-a0fab9437889b7f56407eab76fedfb90R15